core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGeneData.java

```diff
@@ -138,7 +170,21 @@ public static void importData(File geneFile) throws IOException, DaoException {
 	            if (genes.size()==1) {
 	                CanonicalGene gene = genes.iterator().next();
 	                if (!genesWithSymbolFromNomenClatureAuthority.containsKey(symbol)) {
-	                    daoGene.addGene(gene);
+	                    //if exists
+	                    CanonicalGene dbGene = daoGene.getGene(gene.getEntrezGeneId());
```

Comment:
since we will be only adding genes and not updating them, this check can be removed and only the `else` can stay